File: C++/Sharma/OOPS/Factorial.cpp
#include <iostream>
using namespace std;
class Factorial
{
    int n;
    int f = 1;

public:
    void get();
    void show();
};

void Factorial::get()
{
    cin >> n;
    for (int i = 1; i <= n; i++)
    {
        f = f * i;
    }
}

void Factorial::show()
{
    cout << f << endl;
}

int main()
{
    Factorial obj;
    obj.get();
    obj.show();

    return 0;
}

File: RandomTopic/JavaConversion/factorial.java
class factorial{
public static void main(String[] args) {

    int n;
    n = f1(5);
    System.out.println(n); 
    }

    static int f1(int n)
    {
        int s;
        if (n==0) {
            return 1;
        }else{
            s = n*f1(n-1);
        }
        return s;
    }
}
